src/service/api.tsx

```diff
@@ -100,7 +100,7 @@ const api = {
 
   async getDataById(id: number) {
     const obj = await axios.get(`movie/${id}?${API_KEY}`);
-    return obj.data.results;
+    return obj.data;
```

Comment:
I'm not sure but maybe you need to describe this change.

Reply:
I agree, you need to describe what the line removed.

Reply:
updated the description with explanation, thanks